indico/modules/events/operations.py

```diff
@@ -278,3 +280,35 @@ def unlock_event(event):
     db.session.flush()
     logger.info('Event %r unlocked by %r', event, session.user)
     event.log(EventLogRealm.event, EventLogKind.change, 'Event', 'Event unlocked', session.user)
+
+
+def create_reviewing_question(event, question_model, wtf_field_cls, form, data=None):
+    new_question = question_model()
+    new_question.no_score = True if wtf_field_cls.name != 'rating' else getattr(form, 'no_score', True)
```

Comment:
`wtf_field_cls.name != 'rating' or getattr(form, 'no_score', True)`

Also, if `form` is a wtform, then the else block is broken, since the attribute is a field, not its data.